src/server.ts

```diff
@@ -38,7 +38,7 @@ newApp(
     // Kubernetes so the proxy has to run on HTTPS to behave correctly.
     if (env === NodeEnvironmentEnum.DEVELOPMENT) {
       const samlKey = container.resolve(SAML_KEY);
-      const samlCert = container.resolve(SAML_CERT);
+      const samlCert = SAML_CERT;
```

Comment:
looks like this alias is not needed, you can use `SAML_CERT` in the next line

Reply:
fixed in 5650e7f